scripts/js/lib/fs.ts

```diff
@@ -16,7 +16,7 @@ import path from "path";
 import { $ } from "zx/core";
 
 export function getRoot() {
-  return path.normalize(`${__dirname}/../../`);
+  return path.normalize(`${__dirname}/../../../`);
 }
```

Comment:
This function is going to need to be reworked in preparation for packaging the repository. I plan to tackle this next week.